packages/ibm-products/src/components/NonLinearReading/NonLinearReading.tsx

```diff
@@ -19,19 +19,45 @@ import { pkg } from '../../settings';
 const blockClass = `${pkg.prefix}--non-linear-reading`;
 const componentName = 'NonLinearReading';
 
-// Default values for props
-const defaults = {
-  theme: 'light',
-};
+type Theme = 'light' | 'dark';
+interface NonLinearReadingProps extends PropsWithChildren {
+  /**
+   * The keyword of the component appears as a pill.
+   */
+  children: ReactNode;
```

Comment:
When you extends from PropsWithChildren, you get the children prop internally typed. So If we are using **PropsWithChildren** then there is no need to explicitly specify **children** type.

Reply:
yea I know but if its internally typed, the children become optional prop. Here the children is mandatory, thats why I specified it explicitly.

Reply:
In that case, it is fine explicitly specify children type and you can skip extending from  PropsWithChildren. I think any one approach is fine.